### PR TITLE
test: add tests for memcached default values

### PR DESCRIPTION
**Pull request by ligurio**

### CODE REVIEW

File: test/common/suite.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[default]
+core = app
+description = memcached common tests with #! using TAP
+script = common.lua
```
Comment (Totktonada) [inline test/common/suite.ini:4]: There is no such Lua file and it is not used for an app server by test-run AFAIU.

File: test/common/defaults.test.lua [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+})
+test:isnt(mc, nil, 'memcached instance object is not nil')
+test:is(is_port_open(mc_port), true, 'memcached instance is started')
+
+test:isnt(mc.listener, nil, 'memcached instance is started')
+test:istable(mc.listener, 'type of instance.listener is a table')
+test:is(mc.uri, tostring(mc_port), 'instance.uri is equal to port number specified by user')
+test:istable(mc.opts, 'type ofinstance.opts is a table in memcached instance object')
+test:is(mc.opts.if_not_exists, false, 'instance.opts.if_not_exists is equal to false by default')
+test:is(mc.opts.protocol, 'negotiation', 'instance.opts.protocol is equal to "binary" by default')
```
Comment (Totktonada) [inline test/common/defaults.test.lua:39]: It is equal to `'negotiation'` as I see.

**Review by Totktonada — APPROVED** [review]
LGTM. Left several small nits. Feel free to push.

File: test/common/suite.ini (hunk 1 above)
Comment (ligurio) [reply test/common/suite.ini:4]: Right, removed.

File: test/common/defaults.test.lua (hunk 2 above)
Comment (ligurio) [reply test/common/defaults.test.lua:39]: Fixed.

File: test/common/defaults.test.lua [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+test:is(mc.opts.uri, '0.0.0.0:11211', 'instance.opts.uri is equal to "0.0.0.0:11211" by default')
+test:is(mc.opts.storage, 'memory', 'instance.opts.storage is equal to "memory" by default')
+test:is(mc.space.engine, 'memtx', 'default space engine is memtx')
+test:is(mc.opts.name, 'memcached', 'instance.opts.name is equal to "memcached"')
+test:is(mc.opts.verbosity, 0, 'instance.opts.verbosity is equal to 0 by default')
+test:is(mc.opts.sasl, false, 'instance.opts.sasl is false by default')
+test:is(mc.opts.expire_enabled, true, 'instance.opts.expire_enabled is true by default')
+test:is(mc.status, 'r', 'instance.status is equal to "r" by default')
+test:is(mc.name, mc_name, 'instance.name is equal to specified by user')
+test:is(mc.space_name, mc_space_name, 'instance.space_name is equal to "memcached_space"')
```
[the remark this answers is not in the document]
Comment (ligurio) [reply test/common/defaults.test.lua:52]: Sure.

```diff
@@ -49,7 +49,7 @@ test:is(mc.opts.sasl, false, 'instance.opts.sasl is false by default')
 test:is(mc.opts.expire_enabled, true, 'instance.opts.expire_enabled is true by default')
 test:is(mc.status, 'r', 'instance.status is equal to "r" by default')
 test:is(mc.name, mc_name, 'instance.name is equal to specified by user')
-test:is(mc.space_name, mc_space_name, 'instance.space_name is equal to "memcached_space"')
+test:is(mc.space_name, mc_space_name, 'instance.space_name is equal to specified name')
 test:isnt(mc.service, nil, 'instance.service is not nil')
 
 os.exit(test:check() and 0 or 1)
```

File: test/common/defaults.test.lua (hunk 2 above)
Comment (ligurio) [reply test/common/defaults.test.lua:39]:
```diff
--- a/test/common/defaults.test.lua
+++ b/test/common/defaults.test.lua
@@ -36,7 +36,7 @@ test:istable(mc.listener, 'type of instance.listener is a table')
 test:is(mc.uri, tostring(mc_port), 'instance.uri is equal to port number specified by user')
 test:istable(mc.opts, 'type ofinstance.opts is a table in memcached instance object')
 test:is(mc.opts.if_not_exists, false, 'instance.opts.if_not_exists is equal to false by default')
-test:is(mc.opts.protocol, 'negotiation', 'instance.opts.protocol is equal to "binary" by default')
+test:is(mc.opts.protocol, 'negotiation', 'instance.opts.protocol is equal to "negotiation" by default')
 test:is(mc.opts.expire_full_scan_time, 3600, 'instance.opts.expire_full_scan_time is equal to 3600 by default')
 test:is(mc.opts.expire_items_per_iter, 200, 'instance.opts.expire_items_per_iter is equal to 200 by default')
 test:is(mc.opts.readahead, 16320, 'instance.opts.readahead is equal to 16320 by default')
```

File: test/common/suite.ini (hunk 1 above)
Comment (ligurio) [reply test/common/suite.ini:4]:
```diff
--- a/test/common/suite.ini
+++ b/test/common/suite.ini
@@ -1,5 +1,4 @@
 [default]
 core = app
 description = memcached common tests with #! using TAP
-script = common.lua
 is_parallel = False
```

**Pull request merged**